routes/__init__.py

```diff
@@ -0,0 +1,22 @@
+from flask import Blueprint, current_app, request
+from membrane.client.flask import membrane_current_user, membrane_login_required
+
+blueprint = Blueprint("main", __name__)
+
+
+@blueprint.before_request
+def log_request_info():
+    current_app.logger.debug("Headers: %s", request.headers)
+    current_app.logger.debug("Body: %s", request.get_data())
+
+
+@blueprint.route("/health", methods=["GET"])
+def health():
+    return "ok", 200
```

Comment:
@RussellJimmies your /health should line up with this one. In general we want constants to be in variables configured in .env, but in this particular /health will evolve in the future to provide dynamic app health information (counters of exceptions HTTP codes 4xx, 3xx and 5xxx counts, etc)